Clarify HTTP request types that do or do not use a body (#31770)
mdn/content: files/en-us/web/http/messages/index.md

### Body

> The final part of the request is its body. Not all requests have one: requests fetching resources like `GET` or `HEAD` usually don't need a body. Requests that send data to the server in order to update it such as `PUT` or `POST` typically require a body containing the data which is to be used in fulfilling the request (for instance HTML form data).

Reviewer: Thanks a lot. Just a minor reword, what do you think?

```suggestion
The final part of the request is its body. Not all requests have one: requests fetching resources like `GET` or `HEAD` usually don't need a body. Requests that send data to the server to create or update a resource, such as `PUT` or `POST` requests, typically require a body with the data used to fulfill the request (for instance, HTML form data).
```

Author: Yup, that's even stronger!
